test/renate/updaters/avalanche/test_avalanche_model_updater.py

```diff
@@ -26,7 +26,7 @@ def get_model_and_dataset(dataset_size: int = 100):  # TODO: remove
 
 @pytest.mark.parametrize("provide_folder", [True, False])
 def test_avalanche_model_updater(tmpdir, provide_folder):
-    model, train_dataset, test_data = pytest.helpers.get_renate_module_mlp_and_data(
+    model, train_dataset, test_data, _ = pytest.helpers.get_renate_module_mlp_data_and_loss(
```

Comment:
why do we use the "and loss" function and then drop the loss?

Reply:
Done.